lighthouse-core/gather/gatherers/form-elements.js

```diff
@@ -85,22 +85,26 @@ function collectFormElements() {
 /* c8 ignore stop */
 
 class FormElements extends Gatherer {
+  /** @type {LH.Gatherer.GathererMeta} */
+  meta = {
+    supportedModes: ['snapshot', 'navigation'],
+  }
+
   /**
-   * @param {LH.Gatherer.PassContext} passContext
+   * @param {LH.Gatherer.FRTransitionalContext} passContext
    * @return {Promise<LH.Artifacts['FormElements']>}
    */
-  async afterPass(passContext) {
+  snapshot(passContext) {
     const driver = passContext.driver;
 
-    const formElements = await driver.evaluate(collectFormElements, {
```

Comment:
nit: revert

when I see this structure I assume it's because there was a time where debugging this was necessary and `console.log(formElements)` seems useful + easy to keep :)